Mathlib/RingTheory/Binomial.lean

```diff
@@ -450,6 +452,54 @@ theorem choose_add_smul_choose [NatPowAssoc R] (r : R) (n k : ℕ) :
   rw [choose_smul_choose (r + k) (n + k) k (Nat.le_add_left k n), Nat.add_sub_cancel,
     add_sub_cancel_right]
 
+end
+
+/-- Pochhammer version of Chu-Vandermonde identity -/
+theorem descPochhammer_smeval_add [Ring R] {r s : R} (k : ℕ) (h: Commute r s) :
+    (descPochhammer ℤ k).smeval (r + s) = ∑ ij ∈ Finset.HasAntidiagonal.antidiagonal k,
```

Comment:
```suggestion
    (descPochhammer ℤ k).smeval (r + s) = ∑ ij ∈ antidiagonal k,
```
and `open Finset` before the theorem. (This also lets you drop various `Finset.` if you prefer.)